test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1638,7 +1643,13 @@ UCS_TEST_P(test_ucp_sockaddr_protocols_err, tag_zcopy_64k_unexp_sync,
 }
 
 UCS_TEST_P(test_ucp_sockaddr_protocols_err, tag_rndv_unexp,
-           "RNDV_THRESH=0")
+           "RNDV_THRESH=0", "RNDV_SCHEME=get_zcopy")
+{
+    test_tag_send_recv(64 * UCS_KBYTE, false, false);
+}
+
+UCS_TEST_P(test_ucp_sockaddr_protocols_err, tag_rndv_unexp_get_scheme,
+           "RNDV_THRESH=0", "RNDV_SCHEME=get_zcopy")
```

Comment:
what is the difference in these tests

Reply:
when we are asking for GET_ZCOPY, but transport doesn't support it (e.g. TCP), it fallbacks to RNDV AM
the idea is to cover as much as possible cases

Reply:
these two tests are identical (this one and the previous one)